Dockerfile

```diff
@@ -101,4 +102,5 @@ USER 1001
 
 ENV LD_LIBRARY_PATH=/open-vds:$LD_LIBRARY_PATH
 ENV OPENVDS_AZURESDKFORCPP=1
+ENV LD_PRELOAD=/usr/lib/libjemalloc.so
```

Comment:
Depending on whether we consider using `LD_PRELOAD` a fix or a workaround, we should consider opening an issue to keep track of the memory allocator. This would make it easier to revisit this once/of OpenVDS implements a different memory allocation strategy or helps us decide if we want to implement the allocator in a different way, e.g., linking OpenVDS against it.